packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -1,182 +1,290 @@
+
+<Tabs marginTop="spacing.8" variant="bordered" orientation="horizontal">
+  <Box position="sticky" top="0px" zIndex={1} backgroundColor="surface.background.gray.intense">
+    <TabList>
+      <TabItem value="basic">1. Basic Setup</TabItem>
+      <TabItem value="fonts">2. Install Fonts</TabItem>
+      <TabItem value="ts">3. TypeScript</TabItem>
+    </TabList>
+  </Box>
+  <TabPanel value="basic">
+    <Heading size="large" marginTop="spacing.7">Add blade to your application</Heading>
+
+    <StepGroup orientation="vertical" size="medium">
+      <StepItem
+        title="1. Install blade as a dependency."
+      >
+        <Text>Blade has a peer dependency on a few libraries, you can skip adding it if you already have it installed in your project.</Text>
+
+        ```shell
+        yarn add @razorpay/blade styled-components@5.3.11 @razorpay/i18nify-js @razorpay/i18nify-react
+        ```
+        <List>
+          <ListItem>
+            <ListItemLink href="https://www.npmjs.com/package/styled-components/v/5.3.11">styled-components@5</ListItemLink>
+            <ListItemText>
+              <Tooltip content="Currently blade only supports version 5">
+                <TooltipInteractiveWrapper position="relative" left="2px" top="2px">
+                  <InfoIcon size="small" />
+                </TooltipInteractiveWrapper>
+              </Tooltip>
+            </ListItemText>
+          </ListItem>
+          <ListItem>
+            <ListItemLink href="https://www.npmjs.com/package/@razorpay/i18nify-js">@razorpay/i18nify-js</ListItemLink>
+            <ListItemText>
+              <Tooltip content="i18n library that blade depends on. Follow their installation guide for more info">
+                <TooltipInteractiveWrapper position="relative" left="2px" top="2px">
+                  <InfoIcon size="small" />
+                </TooltipInteractiveWrapper>
+              </Tooltip>
+            </ListItemText>
+          </ListItem>
+          <ListItem>
+            <ListItemLink href="https://www.npmjs.com/package/@razorpay/i18nify-react">@razorpay/i18nify-react</ListItemLink>
+            <ListItemText>
+              <Tooltip content="A state management React wrapper for @razorpay/i18nify-js that maintains locale state of your page">
+                <TooltipInteractiveWrapper position="relative" left="2px" top="2px">
+                  <InfoIcon size="small" />
+                </TooltipInteractiveWrapper>
+              </Tooltip>
+            </ListItemText>
+          </ListItem>
+        </List>
+
+      </StepItem>
+      <StepItem
+        title="2. Add ReactNative Dependencies [Optional for web projects]"
+      >
+        ```shell
+        yarn add @floating-ui/react-native@0.10.0 react-native-reanimated@3.4.1 react-native-tab-view@3.5.2 react-native-pager-view@6.2.1 react-native-svg@12.3.0 react-native-gesture-handler@2.9.0 @gorhom/bottom-sheet@4.4.6 @gorhom/portal@1.0.1
```

Comment:
This is making the X-axis overflow, 

https://github.com/razorpay/blade/assets/46647141/097f9c3a-9ce0-4801-860d-f1fe6c80fecf

Reply:
Can't do much, storybook forces the codeblock to be fixed width.

Reply:
can you add Box wrapper around that codeblock with fixed width and overflow auto?